vendor/bat-native-ledger/src/bat_publishers.cc

```diff
@@ -140,12 +130,8 @@ ledger::ActivityInfoFilter BatPublishers::CreateActivityFilter(
 
 ledger::ActivityInfoFilter BatPublishers::CreateActivityFilter(
```

Comment:
Before, we were keying from pub_id, month, and year (which normally would return one entry). It seems like some of these filters just key off of pub_id which can return multiples over several contribution months. Do we want to have reconcile_stamp required for these?

Reply:
removed unused filters